### ISSUE TITLE
Documentation issue on Authentication

### ISSUE THREAD

**Issue by DomtronVox** [podio-community/podio-php#168]
On the page https://podio-community.github.io/podio-php/authentication/ for the server side flow section there is a call to Podio::authorize_url(REDIRECT_URI). Actually using this causes errors because authorize_url wants two arguments REDIRECT_URI and a scope string. 

I had to dig through the code and find the function then track down what "scope" was supposed to have in it.

**Comment by carlfredrikhero** [comment]
I believe neither the documentation nor the library is up to date with the (relatively new) scope feature.

I will make an attempt to update the library (and the documentation).

## Areas where scope is needed:

 - [ ] Documentation (see above)
 - [ ] [PodioOauth.php](https://github.com/podio-community/podio-php/blob/master/lib/PodioOAuth.php) According to https://developers.podio.com/authentication/server_side the expected return value is
    ```
    {
        "access_token": ACCESS_TOKEN,
        "token_type": "bearer",
        "expires_in": EXPIRES_IN,
        "refresh_token": REFRESH_TOKEN,
        "scope": GRANTED_SCOPE_STRING,
        "ref":
        {
            "type": "user",
            "id": USER_ID
        }
    }
    ```
 - [ ] https://github.com/podio-community/podio-php/blob/master/lib/Podio.php#L109
 - [ ] `Podio::authorize_url` in https://github.com/podio-community/podio-php/blob/master/lib/Podio.php#L133

**Comment by carlfredrikhero** [comment]
One thing I want to discuss with the other maintainers is how we should treat the scope string that is returned from the api. On https://developers.podio.com/authentication/scopes#granted Podio gives an example `user_2050398:read user_2050398:write app_14956:delete`.

As I see it we have 2 options:

1. Just store the scope string "as is" in the OAuth class.
    **Pros**: Easier to implement in the library, not opinionated.
    **Cons**: The string split logic has to be implemented every time the library is used.

2. We store the scope in a "better" data format.
    **Pros**: Possibly easier to use for the developer that uses the library, but that might be subjective. How do we know that it is better? A simple array wouldn't be enough imho.
    **Cons**: There is quite a big chance that we don't anticipate all use cases and build a solution that don't work for everyone.

I would vote on the "safe" choice here, just store the string and let the developer figure out the what works best for his/her particular case. *If* a clear pattern appears, then we can change the library implementation.

**Comment by daniel-sc** [comment]
@carlfredrikhero is there still something open or can we close this issue?
(hope I didn't merged #169 permaturely..)

**Comment by carlfredrikhero** [comment]
No, this is done. I went with the first option and just store scope as a string.
